src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.hpp

```diff
@@ -210,7 +210,7 @@ class VTKPolyDataWriterInterface
   void clearData();
 
 
-private:
+protected:
```

Comment:
Is the increase of visibility required? I am not understanding from the review what child calls these?

Reply:
You are right. This is needed to subclass PolyDataWriterInterface to export data into a VTK compatible one.